glide.yaml

```diff
@@ -43,3 +43,4 @@ import:
   - tools/clientcmd
 testImport:
 - package: github.com/onsi/gomega
+- package: github.com/onsi/ginkgo/ginkgo
```

Comment:
You are right. Just tried to remove ginkgo and run glide, I thought that would make it download ginkgo. I hope we get a package manager as soon as possible; every single tool works in a different way.

I'll remove this line.